src/app/views/shared/dynamic-library-examples/example-components/highcharts/highcharts.component.ts

```diff
@@ -244,6 +248,14 @@ export class HighchartsComponent implements OnInit {
     });
   }
 
+  private getData__example_3e() {
+    this.getData(MockData.PrikkedeDataMedToSerier, {
+      ...this.diagramOptions,
+      title: this.titles.title_3e,
+      units: undefined,
```

Comment:
trenger man å sette `units: undefined` her?

Reply:
Nei, det har du rett i slik det er nå, men eksemplet er prematurt, og skulle egentlig brukt linjediagram, og da må en sette `units: undefined`. Jeg valgte å skjule eksemplet i stede for å fjerne det, siden vi trenger det i #638